packages/react-examples/src/react-charting/VerticalBarChart/VerticalBarChartPage.tsx

```diff
@@ -48,6 +51,9 @@ export class VerticalBarChartPage extends React.Component<IComponentDemoPageProp
             <ExampleCard title="VerticalBarChart dynamic" code={VerticalBarChartTooltipExampleCode}>
               <VerticalBarChartTooltipExample />
             </ExampleCard>
+            <ExampleCard title="VerticalBarChart Date Axis" code={VerticalBarChartDateLabelExampleCode}>
```

Comment:
You need to add an entry in VerticalBarChart.doc.tsx to show in the docsite.
Also update the supported axis table in docs/TechnicalDetails.md

Reply:
Done